community/dogefetch/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+make install PREFIX=/usr/bin
```

Comment:
`PREFIX=/usr/bin` would install it to `/usr/bin/bin/dogefetch`, no? 
Should just be `PREFIX=/usr`